File: 백준/baekjoon_10250.java
package baekjoon;

// https://www.acmicpc.net/problem/10250

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class baekjoon_10250 {

	public static void main(String[] args) throws NumberFormatException, IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int T = Integer.parseInt(br.readLine());
		
		for(int i = 0; i < T; i++) {
			String[] inputArray = br.readLine().split(" ");
			int H = Integer.parseInt(inputArray[0]);
			int W = Integer.parseInt(inputArray[1]);
			int N = Integer.parseInt(inputArray[2]);
			String Y;
			String X;
			
			// 온 순서를 건물의 층 수로 나눈 나머지가 0일 경우 최상층을 의미
			if(N % H == 0) {
				Y = Integer.toString(H);
				X = Integer.toString((int) N / H);
			}
			else {
				Y = Integer.toString(N % H);
				X = Integer.toString(1 + ((int) N / H));
			}
			
			// X가 한자리일 경우, 앞에 '0'추가
			if(X.length() == 1) {
				X = "0" + X;
			}
			
			System.out.println(Y+X);
		}
	}
}

File: 백준(python)/baekjoon_10250.py
# https://www.acmicpc.net/problem/10250

import sys

T = int(sys.stdin.readline().strip())

for _ in range(T):
    inputList = list(map(int, sys.stdin.readline().strip().split()))
    H = inputList[0]
    W = inputList[1]
    N = inputList[2]
    
    # 온 순서를 건물의 층 수로 나눈 나머지가 0일 경우 최상층을 의미
    if N % H == 0:
        Y = str(H)
        X = str(N // H)
    else:
        Y = str(N % H)
        X = str(1 + (N // H))

    # X가 한자리일 경우, 앞에 '0'추가
    if len(X) == 1:
        print(f'{Y}0{X}')
    else:
        print(f'{Y}{X}')